packages/contracts/hardhat.config.ts

```diff
@@ -96,19 +110,10 @@ const config: HardhatUserConfig = {
   },
   defaultNetwork: 'hardhat',
   networks: {
-    hardhat: {
-      chainId: 1337,
-      loggingEnabled: false,
-      gas: 12000000,
-      gasPrice: 'auto',
-      blockGasLimit: 12000000,
-      accounts: {
-        mnemonic: 'myth like bonus scare over problem client lizard pioneer submit female collect',
-      },
-      mining: {
-        auto: true,
-        interval: 0
-      }
+    hardhat: {...default_config},
+    compose: {
+      ...default_config,
+      url: 'http://hardhat:8545'
```

Comment:
I don't like the idea that `hardhat.config.ts` has to be `docker-compose` aware, but I understand it may not be easy to move all configuration logic inside `extended.config.ts`.

Reply:
I agree. 
To avoid this, I've reused the same extension pattern used for the network contracts.

Fixed in d20e602.